### PR TITLE
Update ci builds

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,15 +1,58 @@
...
+
+if("${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}" GREATER_EQUAL 3.9.2)
+   # Set and use the newest cmake policies that are available, and have been
+   # verified to work
+  cmake_policy(VERSION 3.13.1) # As of 2018-12-04 GDCM builds with cmake version 3.13.1 new policies.
+endif("${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}" GREATER_EQUAL 3.9.2)
+
+# GDCM version 3.0.0 will only support C++11 and greater
+if(CMAKE_CXX_STANDARD EQUAL "98" )
+   message(FATAL_ERROR "CMAKE_CXX_STANDARD:STRING=98 is not supported in ITK version 5 and greater.")
```
Comment (dzenanz) [inline CMakeLists.txt]: Copy-paste mistake: we should not mention ITK here.

File: .travis.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ before_install:
 install: true
 before_script: true
```
Comment (dzenanz) [inline .travis.yml]: in commit message "one translation until" -> "one translation unit"

File: CMakeLists.txt (hunk 1 above)
Comment (hjmjohnson) [reply CMakeLists.txt]: Fixed.  Thanks.

File: .travis.yml (hunk 2 above)
Comment (hjmjohnson) [reply .travis.yml]: Fixed.

**Comment by malaterre** [comment]
I simply cherry-picked commit 05fc8fbba9cf73ead6ee7e602cd549ba0762b7df (3acab7a3d7a0475581d9e3ccd2ebda65cfe35d68)

**Comment by hjmjohnson** [comment]
@hjmjohnson Thanks.

**Pull request closed without merging**